cmd/release/main.go

```diff
@@ -242,6 +244,21 @@ func (c *command) calculateNewReleasePlugins(currentRelease *release.PluginRelea
 	return plugins, nil
 }
 
+func pluginDependencies(plugin *plugin.Plugin) []string {
+	if len(plugin.Deps) == 0 {
+		return nil
+	}
+	deps := make([]string, len(plugin.Deps))
+	for i, dep := range plugin.Deps {
+		if dep.Revision != 0 {
+			log.Fatalf("unsupported plugin dependency revision: %v", dep.Revision)
```

Comment:
I wish we hadn't modeled revisions in plugin dependencies - they don't make much sense. This will just ensure we don't introduce them in the future.

Reply:
Maybe. If there are 3 revisions of the same plugin version, and I want to ensure I'm using the same exact docker image, how can this be done without an identifier separate from the plugin version?

Reply:
Using latest probably would have been good enough 🤷 , so I definitely see your point.